src/Datadog.Trace/Agent/IAgentWriter.cs

```diff
@@ -12,6 +13,6 @@ internal interface IAgentWriter
 
         Task FlushAndCloseAsync();
 
-        void OverrideApi(IApi api);
+        void OverrideApiBaseEndpoint(Uri uri);
```

Comment:
Consider renaming to `SetApiBaseEndpoint()`.

Reply:
Good idea, implemented in a8ae6f5ec783d182198779a59a280a26eb12a5d1